src/platform/forms-system/src/js/widgets/PhoneNumberWidget.jsx

```diff
@@ -10,6 +10,11 @@ export default class PhoneNumberWidget extends React.Component {
     super(props);
     this.state = { val: props.value };
   }
+  componentDidUpdate(previousProps) {
```

Comment:
This change had to be made for the prefill to work. I'm not sure why this widget can get prefilled on one of our standard form apps but not VAOS. 🤔